frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -59,7 +59,13 @@ Cypress.Commands.add('testA11y', (target: string) => {
 export const checkErrors = () =>
   cy.window().then((win) => {
     if (win.windowError) {
-      throw new Error(`window/js runtime error detected: ${win.windowError}`);
+      throw new Error(
```

Comment:
Does `throw win.windowError` work instead of creating a new `Error`? Or adding an assertion that `windowError` is nullish?

Reply:
Hmm...I had tried it in the past but not since I truley changed `win.windowError` to be an `Error` not an `Event`.  I'll try to throw it again not that it's a true `Error`

Reply:
`throw win.windowError` results in the following; we lose stack-trace, but since it's minified, not sure how useful it is:
```
ReferenceError: i is not defined
```
As opposed to throwing new Error w/ JSON.stringify & `\n` processing:
```
     window error detected: {
 "message": "i is not defined",
 "stack": "ReferenceError: i is not defined
    at http://localhost:9000/static/main-35bf7a55e8c0461351e2.js:102081:5
    at renderWithHooks (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:371358:18)
    at mountIndeterminateComponent (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:373592:13)
    at beginWork$1 (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:374736:16)
    at HTMLUnknownElement.callCallback (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:356597:14)
    at Object.invokeGuardedCallbackDev (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:356647:16)
    at invokeGuardedCallback (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:356704:31)
    at beginWork$$1 (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:379467:7)
    at performUnitOfWork (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:378458:12)
    at workLoopSync (http://localhost:9000/static/vendors~main-47b21d89223a2a918b47.js:378435:22)"
}
```